pilot_data_analysis.Rmd

```diff
@@ -309,31 +299,32 @@ df_t_first_aoi %>%
 
 ## Mixed-level analysis: First look location
 
-```{r first_look_analysis, results = "verbatim"}
+```{r first_look_analysis, results = "verbatim", eval = FALSE}
 first_look_logistic <- glmer(first_look_location ~ trial_num + (1|subid), data = df_t_first_aoi, family = "binomial")
 
 summary(first_look_logistic)
 ```
 
 Mean level. 
 
-```{r}
+```{r, eval = FALSE}
 mean(df_t_first_aoi$first_look_location)
 ```
 
 
 
 
 # Main analysis 2: Differential analysis 
+This analysis explores whether kids looked at the target more than distractor during anticipatory period. 
 
 ## Data visualization across different test trials
 ```{r}
 ms_proportion_trials <- d %>%
-  group_by(t, trial_num) %>%
+  group_by(t, trial_num, experiment_num) %>%
   summarise(target = mean(aoi == "target", na.rm = TRUE),
             distractor = mean(aoi == "distractor", na.rm = TRUE)) %>%
   mutate(proportion_to_target = target / (target + distractor)) %>% 
-  filter(t >= -4000 & t <= 120+ 4000)
+  filter(t >= -500 & t <= 120+ 500)
```

Comment:
Before, we had been using 4s before the pod, which was kind of long. But maybe 500ms is short?

Reply:
Yes, 4sec is too long, but I don't really have a better idea of how long we should expect, I can extend it to 1000ms, but I think 500ms is still good in the sense that even we focus on such a short period of time, the overall evidence suggested that kids looked at target more than distractor pre- and post-POD

@mcfrank , what do you think? Should we set 1000ms or 500ms for the anticipatory window?